index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var Server = require('./lib/server');
+
+module.exports = {
+  server: function (opts) { return new Server(opts); },
+  // Errors:
+  TaskError: require('./lib/errors/task-error'),
+  TaskFatalError: require('./lib/errors/task-fatal-error'),
+  TaskMaxRetriesError: require('./lib/errors/task-max-retries-error')
```

Comment:
Is there any reason to expose this on `index`? Isn't it only used internally by the `Worker` class?